projects/batfish/src/main/java/org/batfish/main/Batfish.java

```diff
@@ -862,11 +852,9 @@ private Answer compileEnvironmentConfigurations(
       Path indepConfigsDir = envSettings.getDeltaCompiledConfigurationsDir();
       if (deltaConfigurationsDir != null) {
          if (Files.exists(deltaConfigurationsDir)) {
-            answer.append(serializeVendorConfigs(
-                  envSettings.getEnvPath(),
+            answer.append(serializeVendorConfigs(envSettings.getEnvPath(),
                   vendorConfigsDir));
-            answer.append(serializeIndependentConfigs(
-                  vendorConfigsDir,
+            answer.append(serializeIndependentConfigs(vendorConfigsDir,
```

Comment:
and these (the original was not much better):

```java
append(serializeIndependentConfigs(
      vendorConfigsDir, indepConfigsDir));
```

would seem much better, or

```java
append(
      serializeIndependentConfigs(vendorConfigsDir, indepConfigsDir));
```

assuming it fit.